api/clean/cleaner.go

```diff
@@ -0,0 +1,34 @@
+package clean
```

Comment:
Should be in the handlers package of http, I.e. http/handlers/cleaner

Reply:
this was for cleaning up the DB